src/posts/feasibility.md

```diff
@@ -0,0 +1,120 @@
+---
+title: The feasibility of a Pause
+description: Is a pause actually possible?
+---
+
+<!-- does feasibility alludes to some amount of resources being enough for the objective or not? I was going to go with "possibility" for a while but it sounded worse -->
+<!-- I almost didn't put sources at all, ideally that should change -->
+
+AGI is not inevitable. It requires hordes of engineers with million dollar paychecks. 
+It requires a fully functional and unrestricted supply chain of the most complex hardware. 
+It requires all of us to allow these companies to gamble with our future.
+
+We understand where these beliefs come from and that it will not be easy to get a pause on the development of the biggest models. 
+But it is also not easy to make an accurate assessment on how likely it is to happen, and this page seeks to argue that a pause is not as unlikely as some people seem to believe.
+
+Those beliefs are not only overly confident, but they also work as a defeating self-fulfilling prophecy. 
+We shouldn't give up without even trying it. 
+Quite the contrary, given that this could be the best chance for regular people to have an impact in the world and the future of the human race. 
```

Comment:
I'd like to add something like this:

Defeatism is comfortable. Giving up is easy, because if there is nothing that we _can_ do, there is nothing that we _should_ do. It's the easy way out.